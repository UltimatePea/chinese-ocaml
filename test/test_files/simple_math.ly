「：骆言测试程序：简单数学计算：」

设「a」为10
设「b」为3
设「c」为7

设「和」为a + b
设「积」为a * b

打印 『简单数学计算演示』
打印 『10 + 3 = 』
打印 和
打印 『10 * 3 = 』
打印 积