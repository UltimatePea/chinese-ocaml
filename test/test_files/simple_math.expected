简单数学计算演示
10 + 3 = 
13
10 * 3 = 
30